packages/react/src/main-nav/MainNavListItemDropdown.tsx

```diff
@@ -0,0 +1,80 @@
+import { ReactNode } from 'react';
+import { BaseButton } from '../button';
+import { boxPalette, packs } from '../core';
+import {
+	DropdownMenu,
+	useDropdownMenuButton,
+	useDropdownMenuContext,
+} from '../dropdown-menu';
+import { Flex } from '../flex';
+import { ChevronDownIcon, ChevronUpIcon } from '../icon';
+import { localPalette } from './localPalette';
+import { mobileBreakpoint } from './utils';
+
+export type MainNavListDropdown = {
+	label: ReactNode;
+	endElement?: ReactNode;
+	beforeElement?: ReactNode;
+	dropdown: ReactNode;
+};
+
+export function MainNavListItemDropdown(props: MainNavListDropdown) {
+	return (
+		<li>
+			<DropdownMenu popoverPlacement="bottom-end" popoverOffset={-8}>
```

Comment:
is -8 significant here?

Reply:
@dagoof yep we need -8 here so that the dropdown and the button trigger slightly overlap (design team want this)

Reply:
This is also what we do in app-layout